ODIN_II/SRC/netlist_cleanup.cpp

```diff
@@ -60,6 +62,8 @@ void identify_unused_nodes(netlist_t* netlist);
 void remove_unused_nodes(node_list_t* remove);
 void calculate_addsub_statistics(node_list_t* addsub);
 void remove_unused_logic(netlist_t* netlist);
+static void count_node_type(nnode_t* node);
```

Comment:
please comply with the other functions' format and remove static keywords, you would need to call this function outside of this file scope